doc/source/ray-core/fault_tolerance/ray_components.rst

```diff
@@ -0,0 +1,66 @@
+Failure of Ray components
+=========================
+
+Ray components failure could lead to tasks/objects/actors failure. 
+
+GCS failure
+-----------
+
+GCS failure usually means the Ray cluster failure and everything will be lost.
+To make GCS fault-tolerant, HA storage is needed to be used. Ray support using
+Redis as the backend storage. The user needs to set up the storage first and make
+GCS connected to the Redis.
+
+To use this feature, the OS environment ``RAY_REDIS_ADDRESS`` needs to be set to
+the Redis address and ``--redis-password`` needs to be passed when calling ``ray
+start``. For example,
+``RAY_REDIS_ADDRESS=redis_ip:port ray start --head --redis-password PASSWORD``.
+
```

Comment:
To use this feature, set the OS environment  ``RAY_REDIS_ADDRESS``  to
the Redis address,  and supply the  ``--redis-password`` flag  with the password when calling ``ray
start``.